### PR TITLE
update gtest item

### PR DESCRIPTION
**Pull request by Jinp800125**

### CODE REVIEW

**Review by KKyang — APPROVED** [review]
Please change the second commit message

**Comment by jichangjichang** [comment]
Please revise the description for this PR

File: clients/common/cblas_interface.cpp [hunk 1]
```diff
@@ -301,3 +301,280 @@ void cblas_gemm<float, float, float>(hipblasOperation_t transA,
...
+    // cblas does not support hipblasLtHalf, so convert to higher precision float
+    // This will give more precise result which is acceptable for testing
+
+    size_t sizeA = (transA == HIPBLAS_OP_N ? k : m) * size_t(lda);
+    size_t sizeB = (transB == HIPBLAS_OP_N ? n : k) * size_t(ldb);
+
+    host_vector<float> A_float(sizeA), B_float(sizeB);
+
+    hipblaslt_cout << "sizeA: " << sizeA << std::endl;
+    hipblaslt_cout << "size_t(lda): " << size_t(lda) << std::endl;
```
Comment (jichangjichang) [inline clients/common/cblas_interface.cpp:505]: Don't print out debug message by default.

File: clients/common/cblas_interface.cpp [hunk 2]
```diff
@@ -301,3 +301,280 @@ void cblas_gemm<float, float, float>(hipblasOperation_t transA,
...
+                                            int64_t             lda,
+                                            const hip_bfloat16* B,
+                                            int64_t             ldb,
+                                            float               beta,
+                                            float*              C,
+                                            int64_t             ldc,
+                                            const float*              AlphaVec,
+                                            bool                alt)
+{
+    // cblas does not support hip_bfloat16, so convert to higher precision float
```
Comment (jichangjichang) [inline clients/common/cblas_interface.cpp:381]: Why AlphaVec is not used in this function?

File: tensilelite/Tensile/Components/Signature.py [hunk 3]
```diff
@@ -125,8 +127,7 @@ def __call__(self, writer) -> SignatureBase:
 
         if writer.states.needBiasType:
             signature.addArg("biasType",    SVK.SIG_VALUE,        "u32")
-
-        signature.addArg("StrideBias",      SVK.SIG_VALUE,        "u32")
+            signature.addArg("StrideBias",      SVK.SIG_VALUE,        "u32")
```
Comment (KKyang) [inline tensilelite/Tensile/Components/Signature.py:130]: Sould be
```
if kernel["ProblemType"]["UseScaleDVec"] and (kernel["GlobalSplitU"] == 1):
    signature.addArg("StrideBias",      SVK.SIG_VALUE,        "u32")
```

**Review by KKyang — CHANGES_REQUESTED** [review]

**Review by jichangjichang — COMMENTED** [review]
one more thing, Alpha vector in  cublaslt is not called "Scale Alpha Vector".

**Review by KKyang — APPROVED** [review]

**Pull request merged**